pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -50,7 +50,7 @@ <h2 class="my-4 font-bold">1. TRANSACTION DETAILS</h2>
     </section>
     <div class="py-4 flex justify-center items-center gap-x-2">
       {{ if eq .Source "remote" }}
-      <button type="submit" hx-post="/v1/transactions/{{ .ID }}/send" hx-swap="outerHTML" class="p-1 rounded w-36 bg-green-700 font-semibold text-white md:p-2 hover:bg-green-700/80">Accept</button>
+      <button type="submit" hx-post="/v1/transactions/{{ .ID }}/send" hx-swap="none" class="p-1 rounded w-36 bg-green-700 font-semibold text-white md:p-2 hover:bg-green-700/80">Accept</button>
```

Comment:
Since users will be redirected to the `Transaction Detail` page after accepting a transaction, it seemed best to ensure that a swap will not occur.